FRC1076-2016-Competition/src/org/usfirst/frc/team1076/robot/physical/Robot.java

```diff
@@ -127,6 +127,14 @@ public void robotInit() {
 		armFollower.enableBrakeMode(true);
 		armExtendMotor.enableBrakeMode(true);
 		armExtendFollower.enableBrakeMode(true);
+
+		
+		armMotor.ConfigFwdLimitSwitchNormallyOpen(true);
+		armMotor.ConfigRevLimitSwitchNormallyOpen(true);
+		armMotor.enableLimitSwitch(true, true);
+		armExtendMotor.ConfigFwdLimitSwitchNormallyOpen(true);
+		armExtendMotor.ConfigRevLimitSwitchNormallyOpen(true);
+		armExtendMotor.enableLimitSwitch(true, true);
```

Comment:
I'm not sure. Do the follower motors automatically follow the main motor or do we simply set both to be the same. I'm assuming we use both because it's done so everywhere else.

I'll add the follower motors.